README.md

```diff
@@ -1,6 +1,6 @@
-# <img src="https://twiki.cern.ch/twiki/pub/BEABP/Logos/OMC_logo.png" height="28"> 3
+# <img src="https://raw.githubusercontent.com/pylhc/pylhc.github.io/master/docs/assets/logos/OMC_logo.svg" height="28"> 3
 
-[![Cron Testing](https://github.com/pylhc/omc3/workflows/Cron%20Testing/badge.svg)](https://github.com/pylhc/omc3/actions?query=workflow%3A%22Cron+Testing%22)
+[![Test Coverage](https://github.com/pylhc/omc3/workflows/Coverage/badge.svg)](https://github.com/pylhc/omc3/actions/workflows/coverage.yml?query=branch%3Amaster)
```

Comment:
This should specify the master branch or it will show the status of the latest run, which includes PR runs.

Try this: `[![Tests](https://github.com/pylhc/omc3/actions/workflows/coverage.yml/badge.svg?branch=master)](https://github.com/pylhc/omc3/actions/workflows/coverage.yml)`